app/helpers/kadmin/bootstrap_helper.rb

```diff
@@ -34,5 +34,29 @@ def dropdown(prompt, links = [])
 
       return content_tag(:div, button + list, class: 'dropdown', style: 'display: inline-block;')
     end
+
+    def help_assitance(label, required, title, message)
```

Comment:
Did you mean `assistance`? Also, adding a YARD method documentation would help to describe what this helper is (see other methods in this file for a YARD doc example).